Deploy/IaC/bot.tf

```diff
@@ -165,10 +165,7 @@ resource "azurerm_cognitive_account" "LUISAuthoring" {
   resource_group_name = azurerm_resource_group.GeoBot.name
   kind                = "LUIS.Authoring"
 
-  sku {
-    name = "F0" // S0 not available
-    tier = "Free" // Standard not available
-  }
+  sku_name = "F0" // S0 not available
```

Comment:
It is just a question. Is it a default setting to deploy or is it not possible to deploy with S0 sku?

Reply:
Turns out that the **Authoring REST** calls are free of charge and therefore can only be deployed with SKU F0. In contrast to the query **Endpoint REST** calls. They can be created in any available SKU, there is only F0 and S0 available as I know. 
https://github.com/h2floh/GeoDistributedAzureBot/blob/4cef20af17ec63a43dcf1934bb48c8688515e7df/Deploy/IaC/regions.tf#L130